mitmproxy/contentviews/img/metadata.py

```diff
@@ -0,0 +1,30 @@
+from . import png
+
+class Metadata:
+    def __init__():
+        pass
+
+    def get_png_metadata():
```

Comment:
How about we just make this a top-level function - I don't think we need a class for this? :)